src/main/content/_assets/css/header.scss

```diff
@@ -22,99 +22,99 @@
 $color-kabanero:#C43804;
 
 html, body{
-    height: 100%;
+  height: 100%;
 }
 
 main {
-    min-height: 100%;
+  min-height: 100%;
 }
 
 .navbar {
-    margin: 0 auto;
-    z-index: 9999;
-    box-shadow: 0px 5px 10px 0px #9e9e9e;
```

Comment:
@Bazif-Khan thanks for helping out! could you fix the indentation? For this project we're doing 4 spaces (although its still mixed in some places). Looks like your editor changed it to 2 spaces.

Reply:
I have changed the indentation to 4 spaces.